### ISSUE TITLE
installation to merge data with 64 bit excel.

### ISSUE THREAD

**Issue by kokcheeaun** [usnistgov/REFPROP-issues#430]

**Comment by ianhbell** [comment]
What is the question?

**Comment by kokcheeaun** [comment]
Hi Mr Bell,

Thank you for following up on this query.

I am running a 64 bit excel and want to draw data from refprop however I keep getting this error ?( as attached ) .

Is there something I should check on ? what can I do to solve this issue ?

Regards,
Kok Chee aun

**Comment by ianhbell** [comment]
Please attach your screenshot in the github web interface.  Attachments don't come through in email.

**Comment by kokcheeaun** [comment]
![MicrosoftTeams-image (9)](https://user-images.githubusercontent.com/87840693/127089172-43d68f02-d559-4465-8f8e-5aed953eb009.png)

Hi Mr Bell, 

Thank you for responding. i am trying to used Microsoft excel with the refprop and this it the problem i encounter. what can i do to fix this issue ?

Regards,
Kok Chee Aun

**Comment by ianhbell** [comment]
Never

**Comment by ianhbell** [comment]
@EricLemmon  any idea?

**Comment by EricLemmon** [comment]
I don't believe this is a Refprop error, but rather something caused by your system settings.  But that is only a hunch.

Have you carefully checked every item on the list of possible solutions on this webpage:

https://pages.nist.gov/REFPROP-docs/#resolving-problems-linking-with-excel

The fix to nearly every Excel problem we have dealt with is listed on that page.

**Comment by kokcheeaun** [comment]
Hi Mr Lemmon,

Thanks for pointing me to this document, I will check this out immediately.

Regards,
Kok Chee Aun

**Comment by EricLemmon** [comment]
I moved your issue to the Refprop-Wrappers github site.
